core-bundle/src/Csrf/ContaoCsrfTokenManager.php

```diff
@@ -18,8 +18,9 @@
 use Symfony\Component\Security\Csrf\CsrfTokenManager;
 use Symfony\Component\Security\Csrf\TokenGenerator\TokenGeneratorInterface;
 use Symfony\Component\Security\Csrf\TokenStorage\TokenStorageInterface;
+use Symfony\Contracts\Service\ResetInterface;
 
-class ContaoCsrfTokenManager extends CsrfTokenManager
+class ContaoCsrfTokenManager extends CsrfTokenManager implements ResetInterface
```

Comment:
you must tag the service with `kernel.reset`, there is no autoconfig on `ResetInterface` afair

Reply:
Wanted to do that, but forgot 🙈

Reply:
But it should work? https://github.com/symfony/symfony/blob/582f572f5975d05b1dedbfcc55bc09abba463e5c/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L420-L421

Reply:
Done in e02daa5e937f9c17a002bce79319c7fb1e06f6c9